app/Models/BillHistoryTimestamps.php

```diff
@@ -2,10 +2,56 @@
 
 namespace App\Models;
 
+use App\Models\LegiScan\Bill;
+use App\Notifications\BillHasProgressed;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Notification;
 
 class BillHistoryTimestamps extends Model
 {
     use HasFactory;
+
+    /**
+     * The "booted" method of the model.
+     *
+     * @return void
+     */
+    protected static function booted()
+    {
+        static::created(function (BillHistoryTimestamps $billHistoryTimestamps)
+        {
+            $bill_history = $billHistoryTimestamps->bill_history;
```

Comment:
Oh, ok. The variable name should definitely be `$billHistory`.

For the relation, I thought since the relation was used like a property, then it would look like a property.

Changing it now, though!